### PR TITLE
Allow read only access for schedules

### PR DESCRIPTION
**Pull request by rlaszlo**
Removed the login_required decorator from the schedule view
and check for the read only access settings.

The edit buttons and the content visible only for the logged in
users is also wrapped in conditions.

Added the open graph meta tags like in the workout view.

### CODE REVIEW

**Comment by rlaszlo** [comment]
Hi Roland,

I created this pull request for the issue #124.
I'm quiet new to open source so if you see any problems with the request then please tell.

I hid the "Adding workouts" section too from the template on anon access, because it does not makes any sense for users not logged in.

File: wger/manager/views/schedule.py [hunk 1]
```diff
@@ -84,6 +88,8 @@ def view(request, pk):
 
     template_data['uid'] = uid
     template_data['token'] = token
+    template_data['is_owner'] = is_owner
+    template_data['owner_user'] = user
```
Comment (rolandgeider) [inline wger/manager/views/schedule.py]: A little thing, you need to add ``template_data['show_shariff'] = is_owner`` so the sharing buttons are rendered in the template

**Comment by rolandgeider** [comment]
Hi @rlaszlo ! Thanks, this looks nice, I made a small comment, but then it looks that we can merge this.

**Comment by rolandgeider** [comment]
I always forget, add yourself to AUTHORS.rst as well :)

File: wger/manager/views/schedule.py (hunk 1 above)
Comment (rlaszlo) [inline wger/manager/views/schedule.py]: Thanks, didn't notice it in the workout. Fixing in the next commit.

**Comment by rlaszlo** [comment]
Thank you!

I forgot to check the tests, so the build failed. Fixed it now.

**Comment by rlaszlo** [comment]
Hi Roland,

the AUTHORS.rst is in conflict, so github can resolve it. Should I merge your master branch and push a mergable request?

**Comment by rolandgeider** [comment]
Hi @rlaszlo I just merged this, the conflict in the authors file was trivial so we would be spending more time talking about it than doing it :)

**Pull request merged**